### PR TITLE
Updated comment for channel participation API

### PR DESCRIPTION
**Pull request by jcastrence**
#### Type of change

- Documentation update

#### Description

- Updated comment for channel participation API in orderer.yaml sampleconfig.

#### Related issues

- [FAB-18303](https://jira.hyperledger.org/browse/FAB-18303)

### CODE REVIEW

**Comment by btl5037** [comment]
You didn't introduce it, but it seems counterintuitive to me that the channel participation endpoint is a subset of the Admin server, but its config exists outside of the Admin's config block, while making use of parts of the Admin's configuration. If they are going to use a common config, and one is a subset of the other, my opinion is the channel participation should have been under the Admin config block.

Just an observation, your changes are good!

**Comment by lindluni** [comment]
@wlahti Is this a flake in the Channel Participation integration test:

https://dev.azure.com/Hyperledger/Fabric/_build/results?buildId=24428&view=logs&j=354d5075-cb32-57b3-dae3-b45dd635f928&s=911c1fb9-edef-59d0-0b79-3095b59d26c7&t=c61c7ac1-26da-587e-152c-05fdf7b7a34d&l=4087

File: sampleconfig/orderer.yaml [hunk 1]
```diff
@@ -369,8 +369,8 @@ Admin:
 #   Channel participation API Configuration
 #
 #   - This provides the channel participation API configuration for the orderer.
-#   - Channel participation uses the same ListenAddress and TLS settings of the
-#     Operations service.
+#   - Channel participation uses the ListenAddress and the TLS settings of the
```
Comment (wlahti) [inline sampleconfig/orderer.yaml]: Nit (and I know I overlooked this in the JIRA so my bad) 
```suggestion
#   - Channel participation uses the ListenAddress and TLS settings of the
```

**Comment by wlahti** [comment]
> @wlahti Is this a flake in the Channel Participation integration test:
> 
> https://dev.azure.com/Hyperledger/Fabric/_build/results?buildId=24428&view=logs&j=354d5075-cb32-57b3-dae3-b45dd635f928&s=911c1fb9-edef-59d0-0b79-3095b59d26c7&t=c61c7ac1-26da-587e-152c-05fdf7b7a34d&l=4087

Definitely looks like it. I'll open a JIRA. edit: [FAB-18305](https://jira.hyperledger.org/browse/FAB-18305)

**Review by wlahti — APPROVED** [review]
Looks good. Thanks!

**Review by lindluni — APPROVED** [review]

**Pull request merged**